pyproject.toml

```diff
@@ -10,9 +10,6 @@ requires = [
 testpaths = ["tests"]
 addopts = """
 -v
---cov=ansys.fluent
---cov-report=xml:cov_xml.xml
---cov-report=html
```

Comment:
Moved these to Makefile so that debugging from IDE works (https://pytest-cov.readthedocs.io/en/latest/debuggers.html)